geps/gep-1713/index.md

```diff
@@ -0,0 +1,433 @@
+# GEP-1713: Standard Mechanism to Merge Multiple Gateways
+
+* Issue: [#1713](/kubernetes-sigs/gateway-api/issues/1713)
+* Status: Provisional
+
+(See status definitions [here](overview.md#status).)
+
+## tl;dr
+
+The `Gateway` Resource is a point of contention since it is the only place to attach listeners with certificates. We propose a new resource called `ListenerSet` to allow a shared list of listeners to be attached to a single `Gateway`.
+
+## Goals
+- Define a mechanism to merge listeners into a single `Gateway`
```

Comment:
This GEP is missing guidelines for when Listeners are "compatible" or can safely be merged together, is that meant to be part of this GEP or is it a non-goal?

Reply:
tbh I think that one of the best things about this proposal is that we can let go of the compatible language, since we're being explicit on the fact that Listeners must be _distinct_ across ListenerSets - meaning that the combination of the primary key fields (port, protocol, tls details, and, dependent on protocol, hostname) must be unique.

Practically, to be compatible, Listeners used to have to be distinct anyway, so I think that having the additional complexity of allowing merging has not bought us anything except confusion - quite a few people have asked me about what the difference between "distinct" and "compatible" is.

Reply:
It's hard for me to know what action to take here - it seems like this discussion is beyond this GEP and applies to regular Gateway listeners.

Reply:
resolving since Nick's listener compat PR merged